src/app/pages/account/edit/[username].vue

```diff
@@ -306,17 +386,32 @@ const removeProfilePicture = async () => {
 
 // description
 const descriptionLengthMaximum = 500
-const isEditing = ref<boolean>()
+const isEditingDescription = ref<boolean>()
 const editableDescription = ref<string>()
-const toggleEdit = () => {
-  if (!isEditing.value) {
+const toggleEditDescription = () => {
+  if (!isEditingDescription.value) {
     editableDescription.value = account.value?.description?.trim() || ''
   }
-  isEditing.value = !isEditing.value
+  isEditingDescription.value = !isEditingDescription.value
 }
-const cancelEdit = () => {
+const cancelEditDescription = () => {
   editableDescription.value = account.value?.description?.trim() || ''
-  isEditing.value = false
+  isEditingDescription.value = false
+}
+
+// imprint
+const imprintLengthMaximum = 500
+const isEditingImprint = ref<boolean>()
+const editableImprint = ref<string>()
+const toggleEditImprint = () => {
+  if (!isEditingImprint.value) {
+    editableImprint.value = account.value?.imprint?.trim() || ''
+  }
+  isEditingImprint.value = !isEditingImprint.value
+}
+const cancelEditImprint = () => {
+  editableImprint.value = account.value?.imprint?.trim() || ''
+  isEditingImprint.value = false
```

Comment:
Could you extract a component for such an edit field that is then used twice, once for each input, instead of duplicating the code? That way we'll be able to quickly use those editable fields in other places of the app as well.